src/status_im/contexts/chat/messenger/messages/navigation/view.cljs

```diff
@@ -103,13 +104,7 @@
         :background          :blur
         :size                32
         :accessibility-label :back-button
-        :on-press            (fn []
-                               (rf/dispatch [:navigate-back])
-                               ;; In IOS view-id might be incorrect (if screen closed using swipe),
-                               ;; so we can't rely on `:navigate-back` to close the chat.
-                               ;; https://github.com/status-im/status-mobile/pull/21643#issuecomment-248896204451
-                               (when platform/ios?
-                                 (rf/dispatch [:chat/close])))}
+        :on-press            #(rf/dispatch [:navigate-back screen-id])}
```

Comment:
cc @ilmotta, we can get rid of this, as we will not rely on view-id here

Reply:
Will have a look at the WIP PR @Parveshdhull. Thanks for tagging me